tool/src/js/mod.rs

```diff
@@ -29,6 +29,16 @@ pub fn gen_bindings(
     outs.entry("diplomat-runtime.mjs".to_string())
         .or_default()
         .write_str(include_str!("../../templates/js2/runtime.mjs"))?;
+    outs.get_mut("diplomat-runtime.mjs")
+        .unwrap()
+        .write_str(concat!(
+            "export class FFIError extends Error {\n",
+            "	constructor(error_value) {\n",
+            "	super('Error over FFI');\n",
+            "	this.error_value = error_value; // (2)\n",
+            "	}\n",
+            "}\n",
+        ))?;
```

Comment:
Question: Do we still want to use FFIError? I think replacing `throw diplomatRuntime.FFIError` with `throw Error` was a good move, but I can't really see a use case for keeping FFIError around.

Reply:
I don't want to make this change in JS1, so adding this back to the runtime here.

Reply:
Ahhh. Okay! Everything looks good to me then.